contracts/admin/src/lib.rs

```diff
@@ -40,10 +40,10 @@ pub fn instantiate(
 }
 
 #[cfg_attr(feature = "contract-with-bindings", entry_point)]
-pub fn migrate(deps: DepsMut<'_>, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
+pub fn migrate(deps: DepsMut<'_>, _env: Env, _msg: MigrateMsg) -> StdResult<Response> {
```

Comment:
what is the motivation for switching to the cosmwasm error type?

Reply:
Conversion to our error type is not needed as of now. This change was only because of minimization.

Reply:
we do not want to change the interface later when we have to implement the method. As you may see, all methods at this point return StdResult<Response, ContractError>. If you wanna minimize, then define a `type ContractResult<T> = StdResult<T, ContractError>` as we did at the Lease, for example

Reply:
that applies to all occurrences, pls

Reply:
in all contracts